.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
-bundler_args: --without development
-before_script: 
+bundler_args: --without development system_tests
+before_script:
   - bundle exec rake validate_puppet_syntax
```

Comment:
@petems since you removed these tasks from the Rakefile, you'll need to remove them from the travis.yml.